integration_tests/tests/homePage.cy.ts

```diff
@@ -0,0 +1,36 @@
+//  Feature: user views home page
+//    So that I can navigate to the parts of the service that I need
+//    As a user
+//    I want to view the cards on the home page
+//
+//  Scenario: viewing the home page
+//      Given I am logged in
+//      When I visit the home page
+//      Then I see the correct cards
+//      And I see the sign out button
+
+import HomePage from '../pages/homePage'
+import Page from '../pages/page'
+
+context('Home', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+  })
+
+  //  Scenario: viewing the home page
+  it('shows the track progress card', () => {
+    // Given I am logged in
+    cy.signIn()
+
+    //  When I visit the home page
+    HomePage.visit()
+    const page = Page.verifyOnPage(HomePage)
```

Comment:
I like this visit() method that returns the page object, but it looks like both visit and verify on page return a new page object model, I wonder if we need both to do that? 

would be nice to have 

```
const page = HomePage.visit()
page.verifyOnPage()
```

However, looking at the sign in tests, sometimes verify is useful for a different POM after navigation, so maybe useful to have both return the page...